### PR TITLE
Aa mutations endpoint

### LINKED ISSUE #288: Implement `/aa-mutations` returning JSON
Call SILO with the "AminoAcidMutations" action.

### PR DESCRIPTION
**Pull request by fengelniederhammer**
resolves #288

### CODE REVIEW

**Comment by JonasKellerer** [comment]
Missing:
- e2e tests
- schema for return value

File: lapis2/src/main/kotlin/org/genspectrum/lapis/controller/LapisController.kt [hunk 1]
```diff
@@ -584,10 +683,27 @@ private annotation class LapisAggregatedResponse
...
+        Content(schema = Schema(ref = "#/components/schemas/$NUCLEOTIDE_MUTATIONS_RESPONSE_SCHEMA")),
+    ],
 )
 private annotation class LapisNucleotideMutationsResponse
 
+@Target(AnnotationTarget.FUNCTION)
+@Retention(AnnotationRetention.RUNTIME)
+@Operation(
+    description = "Returns a list of mutations along with properties whose proportions are greater than or equal to " +
+        "the specified minProportion. Only sequences matching the specified sequence filters are considered.",
```
Comment (chaoran-chen) [inline lapis2/src/main/kotlin/org/genspectrum/lapis/controller/LapisController.kt:696]: "along with properties" - this sounds like the API would contain information on the properties of the mutations. Maybe write "counts and proportions" instead?

```suggestion
    description = "Returns a list of mutations along with the counts and proportions whose proportions are greater than or equal to " +
        "the specified minProportion. Only sequences matching the specified sequence filters are considered.",
```

File: lapis2/src/main/kotlin/org/genspectrum/lapis/response/LapisResponse.kt [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+import io.swagger.v3.oas.annotations.media.Schema
+
+data class NucleotideMutationResponse(
+    @Schema(
+        example = "sequenceName:G29741T",
+        description = "If the pathogen only contains one sequence then this is: " +
+            "(nucleotide symbol in reference genome)(position in genome)(mutation's target nucleotide symbol)." +
+            "If it has more than one sequence, then the sequence is contained here: " +
+            "(sequenceName):(nucleotide symbol in reference genome)(position in genome)" +
+            "(mutation's target nucleotide symbol)",
```
Comment (chaoran-chen) [inline lapis2/src/main/kotlin/org/genspectrum/lapis/response/LapisResponse.kt:12]:
```suggestion
        description = "If the genome only contains one segment then this is: " +
            "(nucleotide symbol in reference genome)(position in genome)(mutation's target nucleotide symbol)." +
            "If it has more than one segment (e.g., influenza), then the sequence is contained here: " +
            "(sequenceName):(nucleotide symbol in reference genome)(position in genome)" +
            "(mutation's target nucleotide symbol)",
```

**Review by chaoran-chen — APPROVED** [review]
Thanks! I didn't run the code, but it looks good from reading it. I just added two comments on two descriptions.

**Pull request merged**